### ISSUE TITLE
USWDS - Bug: Remove "navigation" text from aria labels for navigation roles

### ISSUE THREAD

**Issue by littleforest** [uswds/uswds#6301]
### Describe the bug

To avoid redundancy, the word "navigation" should be removed from the `aria-label` examples as screen readers will read both the role and the contents of the label. Removing this redundancy in examples will also help users perpetuate the correct usage.

[Documentation reference](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/navigation_role#associated_wai-aria_roles_states_and_properties)

See related issue in uswds-sites repo: https://github.com/uswds/uswds-site/issues/3059

### Steps to reproduce the bug

1. Copy the component code for the [Footer](https://designsystem.digital.gov/components/footer/) into an HTML file.
2. Read with a screen reader, or open the rotor.
3. Note that the screen reader reads "Footer navigation navigation"

<img width="634" alt="Image" src="https://github.com/user-attachments/assets/306c1207-cac2-47b4-b095-eb8fce720896" />

### Expected Behavior

The screen reader should read: "Primary navigation", "Secondary navigation", "Footer navigation", etc. without the redundant "navigation" word.

### Related code

https://github.com/uswds/uswds/blob/95b83cdc637a859e40f38cbc249c7b9e24414ed0/packages/usa-footer/src/_includes/usa-footer-nav-primary--big.twig#L1
https://github.com/uswds/uswds/blob/95b83cdc637a859e40f38cbc249c7b9e24414ed0/packages/usa-footer/src/usa-footer--big/usa-footer--big.json#L7
https://github.com/uswds/uswds/blob/95b83cdc637a859e40f38cbc249c7b9e24414ed0/packages/usa-footer/src/usa-footer.json#L7
https://github.com/uswds/uswds/blob/95b83cdc637a859e40f38cbc249c7b9e24414ed0/packages/usa-footer/src/usa-footer--slim/usa-footer--slim.json#L7
https://github.com/uswds/uswds/blob/95b83cdc637a859e40f38cbc249c7b9e24414ed0/packages/usa-footer/src/_includes/usa-footer-nav-primary.twig#L12
https://github.com/uswds/uswds/blob/95b83cdc637a859e40f38cbc249c7b9e24414ed0/packages/usa-sidenav/src/usa-sidenav.twig#L25
https://github.com/uswds/uswds/blob/95b83cdc637a859e40f38cbc249c7b9e24414ed0/packages/usa-sidenav/src/usa-sidenav.json#L2
https://github.com/uswds/uswds/blob/95b83cdc637a859e40f38cbc249c7b9e24414ed0/packages/usa-sidenav/src/usa-sidenav~one-level.json#L2
https://github.com/uswds/uswds/blob/95b83cdc637a859e40f38cbc249c7b9e24414ed0/packages/usa-sidenav/src/usa-sidenav~two-levels.json#L2
https://github.com/uswds/uswds/blob/95b83cdc637a859e40f38cbc249c7b9e24414ed0/packages/usa-sidenav/src/usa-sidenav~three-levels.json#L2
https://github.com/uswds/uswds/blob/95b83cdc637a859e40f38cbc249c7b9e24414ed0/packages/templates/usa-documentation/includes/_usa-docs-inner.twig#L2
https://github.com/uswds/uswds/blob/95b83cdc637a859e40f38cbc249c7b9e24414ed0/packages/usa-header/src/usa-header.json#L9
https://github.com/uswds/uswds/blob/95b83cdc637a859e40f38cbc249c7b9e24414ed0/packages/templates/usa-documentation/usa-docs.json#L8
https://github.com/uswds/uswds/blob/95b83cdc637a859e40f38cbc249c7b9e24414ed0/packages/usa-header/src/usa-header~megamenu.json#L8
https://github.com/uswds/uswds/blob/95b83cdc637a859e40f38cbc249c7b9e24414ed0/packages/usa-header/src/usa-header--extended/usa-header--extended.json#L8
https://github.com/uswds/uswds/blob/95b83cdc637a859e40f38cbc249c7b9e24414ed0/packages/templates/usa-documentation/includes/_usa-docs-inner.twig#L2
https://github.com/uswds/uswds/blob/95b83cdc637a859e40f38cbc249c7b9e24414ed0/packages/usa-language-selector/src/usa-language-selector--header/usa-language-selector--header.json#L50

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [x] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Bug%22) for duplicate bug reports.

**Comment by alex-hull** [comment]
Moved into needs refinement, to confirm that this is an issue.

**Comment by amycole501** [comment]
Using NVDA I can confirm this on the USWDS site. I hear "Primary navigation, navigation landmark" and "Secondary navigation, navigation landmark". I checked other USWDS sites and they sound similar.
